examples/ffi/crashinfo.cpp

```diff
@@ -21,31 +21,33 @@ static ddog_CharSlice to_slice_string(std::string &s) {
 
 // TODO: Testing on my mac, the tags appear to have the opposite meaning you'd
 // expect
-static ddog_prof_Option_U32 some_u32(uint32_t i) {
-  ddog_prof_Option_U32 rval = {.tag = DDOG_PROF_OPTION_U32_SOME_U32};
+static ddog_crashtracker_Option_U32 some_u32(uint32_t i) {
+  ddog_crashtracker_Option_U32 rval = {.tag = DDOG_CRASHTRACKER_OPTION_U32_SOME_U32};
   rval.some = i;
   return rval;
 }
-static ddog_prof_Option_U32 none_u32() { return {.tag = DDOG_PROF_OPTION_U32_NONE_U32}; }
+static ddog_crashtracker_Option_U32 none_u32() {
+  return {.tag = DDOG_CRASHTRACKER_OPTION_U32_NONE_U32};
+}
 
 struct Deleter {
-  void operator()(ddog_prof_CrashInfo *object) { ddog_crashinfo_drop(object); }
+  void operator()(ddog_crashtracker_CrashInfo *object) { ddog_crashinfo_drop(object); }
```

Comment:
Since we're going to make a backwards-incompatible API change, I think it's worth doing a pass on the naming to match what we've been doing for the rest of the ffi (see #73).

* Should we pick a smaller prefix for the new crate?
* Both structures and functions should probably have similar prefixes?

E.g. for instance (suggestions only :P):

* `ddog_crashtracker_CrashInfo` => `ddog_ctrack_CrashInfo`
* `ddog_crashinfo_drop` => `ddog_ctrack_CrashInfo_drop`

Reply:
`ctrack` is boo :-)
`ddog_crash_CrashInfo` I'd rather see.

Reply:
`crasht` also occurred to me. It'd be nice if it was short, but I can also live with `crashtracker` if we really like it -- no very strong feelings on the color of the shed ;)

Reply:
crasht is fine too

Reply:
`crasht`  sounds good to me.  Thanks for the suggestions :)